README.md

```diff
@@ -114,3 +115,8 @@ THe source of test against JDK. Default is `upstream`. Supported value is [`upst
   - install-jdk: JDK installed by [AdoptOpenJDK/install-jdk](https://github.com/AdoptOpenJDK/install-jdk) | [actions/setup-java](https://github.com/actions/setup-java)
   - github-hosted : pre-installed JDK on github-hosted environment
 
+### openjdk_testRepo
+Openjdk-test git repo, the AQA test. Default is `openjdk-tests:master`. Support developer's personal repo. 
+
+### tkg_Repo
+TKG git repo, the AQA underlining test framework. Support developer's personal repo.
```

Comment:
Reword to: 
TKG git repo, the underlying framework for the AQA test suite.  Parameter can be set to use developer's personal repo.

Reply:
Updated. Thanks!